src/config.rs

```diff
@@ -78,6 +78,12 @@ pub struct Config {
     /// Default: "0.6"
     pub purge_rewrite_garbage_ratio: f64,
 
+    /// Reycle the garbage, that is, stale append / rewrite logs,
+    /// if the size of stale logs lowered than this value.
+    ///
+    /// Default: None
+    pub recycle_garbage_ratio: Option<f64>,
```

Comment:
I think it's a bit complicated. For append queue, `purge_threshold` is enough. We dynamically decide the amount of recycle files = purge_threshold / target_file_size - active_file_count + 1. For rewrite queue, we don't recycle at all.

Reply:
Got it，let me have a look.